Solutions/Cyware/Hunting queries/MatchCywareIntelWatchlistItemsWithCommonLogs.yaml

```diff
@@ -0,0 +1,26 @@
+id: 61c99147-b749-4164-80b1-c4bfa4efa704
+name: Match Cyware Intel Watchlist Items With Common Logs
+description: |
+  'Query to match common security log identifiers with IOCs held by the Cyware Intel watchlist that is created automatically by Cyware'
+requiredDataConnectors: [
+'DeviceProcessEvents'
+]
+tactics:
+  - Command And Control
+  - Execution
+
+relevantTechniques:
+  - T0853
+  - T0863
+query: |
+CommonSecurityLog
+| join kind=inner (_GetWatchlist('CTIX')
+    | where isnotempty(SearchKey)
+) on $left.DestinationIP == $right.SearchKey
+| project TimeGenerated, SourceIP, DestinationIP, DestinationHostName, Activity, ApplicationProtocol, SearchKey=SearchKey, LastUpdatedTimeUTC
+version: 1.0.0
+metadata:
```

Comment:
Hi @AashiqRamachandran, Metadata is not required for hunting queries within solution